packages/shell-bson-parser/package.json

```diff
@@ -55,7 +55,7 @@
     "acorn": "^8.14.1"
   },
   "peerDependencies": {
-    "bson": "^4.6.3 || ^5 || ^6"
+    "bson": "^6.10.3"
```

Comment:
This would be a `feat!` for this specific package, and unfortunately if we marked this `feat!` the publishing bot wouldn't understand that it's restricted to this particular package

Should we just add 7 here?